airflow/contrib/hooks/gcs_hook.py

```diff
@@ -439,6 +440,10 @@ def create_bucket(self,
         self.log.info('Creating Bucket: %s; Location: %s; Storage Class: %s',
                       bucket_name, location, storage_class)
 
+        # Add airflow-version label to the bucket
+        labels = {} or labels
+        labels['airflow-version'] = 'v' + version.replace('.', '-').replace('+', '-')
```

Comment:
I am against this. I know many companies where they have strict policies on labels.

We currently have this for Dataflow or Dataproc jobs I guess which are temporary jobs but buckets are long-lived and generally, companies have stricter policies.

If someone wants to add this labels then can add it but we should not add it by default.

Reply:
This change results from the operator's retrofitting to comply with the integration guide.

> Tracking GCP resources
> To tag and track GCP resources spawned from Airflow, we have been adding airflow specific label(s) to GCP API service calls whenever possible and applicable. For example, we add “airflow-version” label to the dataproc cluster created. We expect new GCP service operators will also add Airflow top-level label “airflow-version” in addition to any service specific labels.

Reply:
Do you think we should rethink the recommendation written in the integration guide?

Reply:
hmm... What's your view? 

If others are happy with this, I don't want to veto or stop this change.

Reply:
I think that nobody cares about having extra labels. Each resource can have many labels and one additional will support the work, but it will not cause trouble. 

If someone debugs their problems, they will be able to do so with additional information from labels. We should think about the fact that users often do not think about future problems. When there is a problem, they look for a solution, and an additional tip can be very helpful.

Labels are added very often if you create resources using google-cloud, so users can even expect that these labels will exist if using a Cloud Composer. I attach a sample instance of Cloud Composer 
<img width="731" alt="Screenshot 2019-09-02 at 20 14 07" src="https://user-images.githubusercontent.com/12058428/64129782-8e83e680-cdbe-11e9-823a-0aacc013692a.png">

So yes. In my opinion, labels are very helpful. This task was created by me 5 days ago: 
https://github.com/PolideaInternal/airflow/issues/265

Reply:
👍

Reply:
You convinced me. Approved